src/i18n/pl_PL.ts

```diff
@@ -0,0 +1,976 @@
+    cellBuildings: [6, 19],
+    cellExtensions: [
+        '6_0',
+        '6_1',
+        '6_2',
+        '6_3',
+        '6_4',
+        '6_5',
+        '6_6',
+        '6_7',
+        '6_8',
+        '6_9',
+        '19_1',
+        '19_2',
+    ],
+    bedBuildings: [4, 21],
+    schoolBuildings: [1, 3, 8],
+    dispatchCenterBuildings: [7],
+    schoolings: {
+        'Fire Station': [
+            {
+                caption: 'Hazmat',
+                duration: '3 Days',
+            },
+            {
+                caption: 'Mobile command',
+                duration: '5 Days',
+            },
+            {
+                caption: 'ARFF-Training',
+                duration: '3 Days',
+            },
+            {
+                caption: 'Swift water rescue',
+                duration: '4 Days',
+            },
+            {
+                caption: 'Ocean Navigation',
+                duration: '5 Days',
+            },
+            {
+                caption: 'Co-Responder Training',
+                duration: '3 Days',
+            },
+        ],
+        'Police': [
+            {
+                caption: 'Police Aviation',
+                duration: '7 Days',
+            },
+            {
+                caption: 'Firearms training',
+                duration: '5 Days',
+            },
+            {
+                caption: 'Dog handling',
+                duration: '5 Days',
+            },
+            {
+                caption: 'Roads Policing Officer Training',
+                duration: '3 Days',
+            },
+        ],
+        'Rescue': [
+            {
+                caption: 'Critical care',
+                duration: '5 Days',
+            },
+        ],
+    },
+    amount: 'Quantity',
+    search: 'Search',
+    alliance: 'Alliance',
+    premiumNotice:
+        'This feature extends a premium feature of the game and is therefore only available for players with a Missionchief game premium account!',
+    credits: 'Credits',
+    close: 'Close',
+    fullscreen: {
+        expand: 'Activate full screen mode',
+        compress: 'Disable full screen mode',
+    },
+    hideTitle: 'Show heading | Hide heading',
+    vehicle: 'Cars | Car | Cars',
+    building: 'Buildings',
+    station: 'Stations | Station | Stations',
+    distance: 'Distance | Distances',
+    vehicleType: 'Vehicle type',
+    fmsReal2Show: {
+        1: 1,
+        2: 2,
+        3: 3,
+        4: 4,
+        5: 5,
+        6: 6,
+        7: 7,
+        9: 9,
+    },
+    buildingIcons: [
+        'fire',
+        'rss',
+        'hospital',
+        'clinic-medical',
+        'graduation-cap',
+        'shield-alt',
+        'helicopter',
+        'graduation-cap',
+        'helicopter',
+        'building',
+        '',
+        'ship',
+        'ship',
+        'fire',
+        'stethoscope',
+        'shield-alt',
+        'clinic-medical',
+        'plane',
+        'shield-alt',
+    ],
+ pois: [

```

Comment:
@esem22 Why you changed it back to English?

Reply:
@esem22 Please look at the [comment](https://github.com/LSS-Manager/LSSM-V.4/pull/293#discussion_r529435738) I placed before. That's the whole list: